ManatiCR/fdi-front
# FDI-51: Create contacto form.

[FDI-51](https://manati.atlassian.net/browse/FDI-51)
-
**Steps to test**
- Ensure the backend is running
- Go to http://localhost:8080/contacto
- Fill all the inputs and click `enviar`
- In here you should be able to see the node created http://fdi.docker/es/admin/fdi/contactos
- Ensure the email is sent, to get the mailhog link: `ahoy docker mailhog-url`
